parsers/ENTSOE.py

```diff
@@ -1261,32 +1350,68 @@ def fetch_wind_solar_forecasts(
     return forcast_breakdown_list.to_list()
 
 
+@refetch_frequency(timedelta(hours=DEFAULT_LOOKBACK_HOURS_REALTIME))
```

Comment:
If it's fetching 120 hours in the past it should not use the default one. It will create a lot of unnecessary requests if we refetch the data.